var/spack/repos/builtin/packages/py-pyfaidx/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPyfaidx(PythonPackage):
+    """pyfaidx: efficient pythonic random access to fasta subsequences"""
+
+    homepage = "http://mattshirley.com/about"
+    url      = "https://files.pythonhosted.org/packages/75/a5/7e2569527b3849ea28d79b4f70d7cf46a47d36459bc59e0efa4e10e8c8b2/pyfaidx-0.5.5.2.tar.gz"
```

Comment:
https://pypi.io/packages/source/p/pyfaidx/pyfaidx-0.5.5.2.tar.gz

Reply:
fixed; just wondering, where did you find the link?